src/PaymentApi.php

```
@@ -67,7 +67,7 @@ public function getPaymentRequest($paymentRequestId)
      */
     public function createPaymentRequest(PaymentRequest $paymentRequest)
     {
-        if ($paymentRequest->exists()) {
+        if ($paymentRequest->id !== null) {
             throw new PaymentRequestAlreadyExistsException();
```

should we throw something like InvalidArgumentException('Creating payment request requires id to of type null');